tests/bytestring-tests.cabal

```diff
@@ -44,6 +54,10 @@ executable prop-compiled
 
 executable regressions
```

Comment:
Shouldn't this also be a testsuite?

Reply:
It hasn't been testable in ages.  It tries to allocate more RAM than fits in an `Int`, but that's no longer possible.